src/Integration/Resources/Strings.resx

```diff
@@ -547,4 +547,8 @@ This is the general format to use when writing errors to output window or when t
   <data name="SolutionIsClosed" xml:space="preserve">
     <value>The operation requires an open solution.</value>
   </data>
+  <data name="UnknownLanguageName" xml:space="preserve">
```

Comment:
Currently, no. But I don't see the problem with adding a resource here to be consistent with everything else.